### ISSUE TITLE
item.get_type() does not recognize webm

### ISSUE THREAD

**Issue by manujchandra** [aerkalov/ebooklib#312]
Hi,

When using item.get_type() on a webm file, it returns an enum of 0 and the file is not recognized as ITEM_VIDEO.

Although I have not tested it, we can also test if webp files are being recognized as images.

The following [epub](https://github.com/IDPF/epub3-samples/releases/download/20230704/cc-shared-culture.epub) can be used for testing.

Thanks.

**Comment by aerkalov** [comment]
Good point. I guess this extensions mapper could be updated a little bit.
